README.md

```diff
@@ -38,3 +38,57 @@ You can then run rubocop with:
   ```sh
   $ rake hydranorth:ci
   ```
+
+## Configuring Shibboleth
```

Comment:
I'll remove this after I do the work to get this working with IST using omniauth-saml instead.